mw-webapp/src/service/WayService.ts

```diff
@@ -1,24 +1,15 @@
-import {DataSnapshot, off, onValue, ref} from "firebase/database";
+import {collection, getDocs} from "firebase/firestore";
+import {querySnapshotToWayDTOConverter} from "src/converter/wayConverter";
 import {db} from "src/firebase";
 import {Way as WayDTO} from "src/model/firebaseCollection/Way";
+import {PathToCollection} from "src/service/PathToCollection";
 
 export class WayService {
 
-  public static onValueFromRealTimeDb(callBack: (data: WayDTO[]) => void) {
-    const wayRef = ref(db, "/ways");
-
-    const handleValueChange = async (snapshot: DataSnapshot) => {
-      const ways: WayDTO[] = snapshot.val();
-      if (ways !== null) {
-        callBack(ways);
-      }
-    };
-
-    onValue(wayRef, handleValueChange);
-
-    return () => {
-      off(wayRef, "value", handleValueChange);
-    };
+  public static async getWays(): Promise<WayDTO[]> {
```

Comment:
why does it returns DTO to business layer?

Reply:
Because right now we don't have converters for return Way and Users. And it's not this task.

Reply:
DOne. Now service layer return DTO models